gravitee-am-ui/src/app/domain/settings/password-policies/domain-password-policies.component.ts

```diff
@@ -27,16 +31,22 @@ import { PasswordPolicy } from './domain-password-policies.model';
 })
 export class PasswordPoliciesComponent implements OnInit {
   domain: any = {};
+  canDelete: boolean = false;
 
   constructor(
     private route: ActivatedRoute,
+    private authService: AuthService,
+    private dialogService: DialogService,
+    private snackbarService: SnackbarService,
     private passwordPolicyService: PasswordPolicyService,
   ) {}
+
   rows: PasswordPolicy[] = [];
 
   ngOnInit() {
     this.domain = this.route.snapshot.data['domain'];
```

Comment:
There is a need to operate on copy instead of reference. Please update only the required data.